src/backend/libs/efiling-lookup-client/src/test/java/ca/bc/gov/open/jag/efilinglookupclient/DemoLookupServiceImplTest.java

```diff
@@ -6,20 +6,22 @@
 
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
-@DisplayName("Efiling Lookup Client Test Suite")
+@DisplayName("Demo Efiling Lookup Client Test Suite")
 public class DemoLookupServiceImplTest {
 
-    private DemoLookupServiceImpl sut;
-
-    @BeforeAll
-    public void setUp() { sut = new DemoLookupServiceImpl(); }
+//    private DemoLookupServiceImpl sut;
```

Comment:
Do we need these commented out lines in the future?

Reply:
Nope, we don't... nice catch.